pyproject.toml

```diff
@@ -40,9 +40,7 @@ test = [
     "pytest-timeout",
     "torch",
     "torchvision",
-    "tensorflow==1.15",
-    "keras==2.3.1",
-    "protobuf<=3.20",  # https://protobuf.dev/news/2022-05-06/#python-updates
+    "protobuf",  # https://protobuf.dev/news/2022-05-06/#python-updates
```

Comment:
```suggestion
    "protobuf",
```

Reply:
is this even still in use? I believe this was for TF only

Reply:
Removed